script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -26,7 +26,7 @@ Splash, that bone-crushing force is astonishing.
 [sha:1c236f0363eebe3fc4d5e703d8d1ae345860f550]{
 -- Page 10013, Offset 19324.
 -- なまくらの<刃|は>は、その重量だけで、逃げ惑う獲物の頭蓋骨を叩き潰した。
-The dulled blade, by its sole weight, smashes through the skulls of its scampering preys.
+The dulled blade, simply with its weight, smashes through the skulls of its scampering prey.
```

Comment:
The dulled blade smashes through the skulls of its scampering prey with its weight alone.